README.md

```diff
@@ -4,152 +4,183 @@
 
 > Easily enable fast AJAX navigation on any website (using pushState() + XHR)
 
-Pjax is **a standalone JavaScript module** that uses
-AJAX (XmlHttpRequest) and
-[pushState()](https://developer.mozilla.org/en-US/docs/Web/Guide/API/DOM/Manipulating_the_browser_history)
-to deliver a fast browsing experience.
+Pjax is **a standalone JavaScript module** that uses [AJAX](https://developer.mozilla.org/en-US/docs/Web/Guide/AJAX) (XmlHttpRequest) and [pushState()](https://developer.mozilla.org/en-US/docs/Web/Guide/API/DOM/Manipulating_the_browser_history) to deliver a fast browsing experience.
 
-_It allows you to completely transform the user experience of standard websites
-(server-side generated or static ones) to make them feel like they are browsing an app,
-especially for users with low bandwidth connection._
+_It allows you to completely transform the user experience of standard websites (server-side generated or static ones) to make users feel like they are browsing an app, especially for those with low bandwidth connections._
 
 **No more full page reloads. No more multiple HTTP requests.**
 
 _Pjax does not rely on other libraries, like jQuery or similar. It is written entirely in vanilla JS._
 
 ## Installation
 
-- You can install Pjax from **npm**:
+* You can install Pjax from **npm**:
```

Comment:
Is there a reason to change all the bulleted lists from `-` to `*`? They're rendered the same anyway.

Reply:
>the original document had both - and * for bullets, so I just converted one to the other.

Ok, I didn't realize that.

I'm not sure which one is more intuitive.

@robinnorth ?

Reply:
I normally use `-` in Markdown lists, so that would be my preference.

Reply:
I changed the `*` to be `-`